server/etcdserver/api/membership/storev2.go

```diff
@@ -95,11 +96,13 @@ func mustSaveMemberToStore(lg *zap.Logger, s v2store.Store, m *Member) {
 
 func mustDeleteMemberFromStore(lg *zap.Logger, s v2store.Store, id types.ID) {
 	if _, err := s.Delete(MemberStoreKey(id), true, true); err != nil {
-		lg.Panic(
-			"failed to delete member from store",
-			zap.String("path", MemberStoreKey(id)),
-			zap.Error(err),
-		)
+		if !strings.Contains(err.Error(), "Key not found") {
```

Comment:
so far only found this one ("key not found") that needs to be absorbed - because we start with empty store now. I will keep an eye out for any other failures that can be expected.